src/main/resources/application.yaml

```diff
@@ -152,7 +157,13 @@ springdoc:
     path: /swagger-ui
   writer-with-default-pretty-printer: true
 
-event.queue.name: event.queue
+application:
+  kafka:
+    listener:
+      events:
+        concurrency: 1
+        topic-pattern: (${ENV:folio}\.)?(.*\.)?workflow.events
+        group-id: ${ENV:folio}-mod-camunda-workflow-events-group
```

Comment:
Is this saying the environment variable `folio`, all lower case?

This probably should be a normal environment variable and then be more descriptive, such as `FOLIO_NAMESPACE` (I randomly chose the word `NAMESPACE` in this example).

Reply:
This was following implementation details and properties conventions from mod-remote-storage.

Reply:
I see I misread that.
`folio` is the value and not the environment name.
`ENV` is the environment variable!?
I think that is extremely bad practice and we should not reproduce bad practice.

This should be something other than the word `ENV`.
There is too much namespace conflict potential.

I now strongly ask for `FOLIO_ENV` or `ENV_FOLIO`.

Reply:
No argument here. But would prefer it be a follow issue to fully document and test.